funnel/models/profile.py

```diff
@@ -16,6 +17,27 @@ class Profile(ProfileBase, db.Model):
 
     description = MarkdownColumn('description', default=u'', nullable=False)
 
+    parent_spaces_test = db.relationship('ProposalSpace',
+        lazy='dynamic',
+        primaryjoin="and_(ProposalSpace.profile_id == Profile.id, ProposalSpace.parent_space_id == None)"
+        )
```

Comment:
@jace need some help with this one, `parent_spaces` below is returned by the `fetch_sorted()` classmethod which, along with sorting the results, [does a state filter](https://github.com/hasgeek/funnel/blob/f29f04004f348de3f8309a9f6b41927de7aa95d6/funnel/models/space.py#L379-L389), which I can't do where without importing `ProposalSpace`, which I can do as I removed the circular import issue from space.py, but want to check if it's possible without importing `ProposalSpace`

Reply:
That entire function could be a SQL expression, which means it could be a relationship.

Reply:
Leave this aside for now. I'll look at rewriting it as a relationship later.